bdr-veeam/veeam-s3-copy-job-diag.ps1

```
@@ -0,0 +1,143 @@
+} catch { Write-Host "  FAILED: $_" }
+
+Write-Host ""
+Write-Host "=========================================="
+Write-Host "3: Try New-VBRBackupCopyJobStorageOptions (minimal)"
+Write-Host "=========================================="
+try {
+    Write-Host "  Trying with just -EnableEncryption -EncryptionKey..."
+    $OPTS = New-VBRBackupCopyJobStorageOptions -EnableEncryption -EncryptionKey $KEY
+    Write-Host "  [OK] $($OPTS.GetType().FullName)"
+    $OPTS | Get-Member -MemberType Property | ForEach-Object {
+        $PN = $_.Name; try { Write-Host "    $PN = $($OPTS.$PN)" } catch {}
```

PSScriptAnalyzer (1.24.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)